### PR TITLE
add goto home button

### LINKED ISSUE #411: Add "Goto Home" button after completing the join section
# Add "Goto Home" button after completing the join section

**Description**

After completing the join section, the user will see the copy button and the user will click the copy button to get the link and send it to Ankush on Twitter. After that, Ankush will send the invite link to the user. After that, the user will click Go to Home button for redirect to home page.

**After changes**

216828651-6180e31d-81ec-4ad7-be3d-4dbe6a9b5d4d

**Note: - Button text will be "Goto Home"**

### PR DESCRIPTION
**Pull request by sahsisunny**
### What is the change?

1. `join.html` - Added "Goto Home" button in end of the join section.
2. `css/join.css` - Added css for "Goto Home" button.

### Is Development Tested?

- [x] Yes
- [ ] No

### Before Change Screenshots
image

### After Change Video

https://user-images.githubusercontent.com/70854507/223294887-a5af8f1e-3c6c-43d7-9deb-a5931142aecf.mp4

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -407,6 +407,11 @@ <h2 class="progress">Here is your personalized link</h2>
               joining link within 7 working days.
             </li>
           </ol>
+          <div class="goto-home-button-container">
+            <a class="button-filled goto-home-btn" href="./goto.html"
```
Comment (khushi818) [inline join.html]: Just a minor suggestion ,we have a class button-filled but it is not a filled button. It should be button-outline

Comment (sahsisunny) [reply join.html]: yes @khushi818 thanks

**Review by khushi818 — APPROVED** [review]
LGTM^_^

**Review by Maheima — APPROVED** [review]
LGTM

**Pull request merged**